lib/fork.js

```diff
@@ -36,7 +37,11 @@ module.exports = (file, opts, execArgv) => {
 		} : false
 	}, opts);
 
-	const args = [JSON.stringify(opts), opts.color ? '--color' : '--no-color'];
+	const args = [JSON.stringify(opts)];
+
+	if (hasFlag('--color') || hasFlag('--no-color')) {
+		args.push(`--color=${opts.color}`);
```

Comment:
Shouldn't this forward `--no-color` if specified? And forward `--color` exactly as it was received?

Reply:
To forward them *exactly* like they were received requires some sort of parsing. We can forward `--no-color` instead of `--color=false` if it was specified though.